google/cloud/forseti/common/gcp_type/iam_policy.py

```diff
@@ -342,12 +342,17 @@ def matches(self, other):
         else:
             other_member = IamPolicyMember.create_from(other)
 
-        # Bucket IAM supports a special "allUsers" member, whose value is simply
-        # "allUsers", without a colon separator and a second fragment.
+        # Bucket IAM supports a special "allUsers" and "allAuthenticatedUsers"
+        # members, whose value is simply "allUsers" and "allAuthenticatedUsers",
+        #  without a colon separator and a second fragment.
```

Comment:
Remove the space before `without`, so that it aligns the the previous comment line

Reply:
Done